dist/fullpage.js

```diff
@@ -2809,7 +2809,7 @@
 
             removeClass(element, NO_TRANSITION);
             return css(element, {
-                '-webkit-transition': transition,
+                'webkitTransition': transition,

```

Comment:
this change looks like it's removing a vendor prefix here !? :
'-webkit-transition'

Soes not feel a "safe" change to me...